examples/structured_2d_dgsem/elixir_advection_basic_coupled.jl

```diff
@@ -0,0 +1,87 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+
+###############################################################################
+# semidiscretization of the linear advection equation
+
+advectionvelocity = (1.0, 1.0)
+equations = LinearScalarAdvectionEquation2D(advectionvelocity)
+
+# Create DG solver with polynomial degree = 3 and (local) Lax-Friedrichs/Rusanov flux as surface flux
+solver = DGSEM(polydeg=3, surface_flux=flux_lax_friedrichs)
+
+# First mesh is the left half of a [-1,1]^2 square
+coordinates_min = (-1.0, -1.0) # minimum coordinates (min(x), min(y))
+coordinates_max = ( 0.0,  1.0) # maximum coordinates (max(x), max(y))
+
+cells_per_dimension = (8, 16)
+
+mesh = StructuredMesh(cells_per_dimension, coordinates_min, coordinates_max)
```

Comment:
Maybe use `mesh1`, `cells_per_dimension1` etc. such that it can be changed from the outside with `trixi_include`?